internal/testing/mock/mock_environment.go

```diff
@@ -104,7 +104,7 @@ func CreateMockEnvironment(ctx context.Context, kClient *fake.Clientset) (*MockE
 	}
 
 	noAddressesVM := getDefaultVMSpec(MockVMNameNoAddresses, cluster)
-	noAddressesVM.Status.Resources.NicList = []*prismClientV3.VMNicOutputStatus{}
+	noAddressesVM.Status.Resources.NicList = nil
```

Comment:
(As an aside, I'd like to figure out if this response was incorrectly mocked, or if the Prism Central API changed since it was mocked).